packages/Overlay/src/Overlay.js

```diff
@@ -109,7 +112,7 @@ const Overlay = props => {
           onExited={onAfterClose}
         >
           {state => (
-            <sc.Overlay data-pka-anchor="overlay" {...moreProps} onKeyDown={handleEscKey}>
+            <sc.Overlay id="overlay" data-pka-anchor="overlay" {...moreProps} onKeyDown={handleKeyDown}>
```

Comment:
🤔 could be possible to be more than one overlay in the same page??

Reply:
I'll check how this works when we have a Panel open as well as Modal showing, there could be an issue there, so will create a story for that case